lib/elastic_apm/agent.rb

```diff
@@ -91,8 +91,11 @@ def initialize(config)
 
     # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
     def start
-      unless config.disable_start_message
-        info '[%s] Starting agent, reporting to %s', VERSION, config.server_url
+      unless config.disable_start_message?
+        puts format(
```

Comment:
Rails' logger isn't sending to `STDOUT` at this point and most devs (in my experience) don't watch `log/development.log` close enough to notice that it went there and not into their `server` process.

Using `puts` is a way around it but as some people hate libraries writing _anything_ to stdout I added the option back in the day to disable it.

Reply:
This has come up a few times actually and I'm always leaning more in the way of helping users onboard. But I'm curious if you think I'm in the wrong.

Reply:
It spams the test output now so perhaps this is for another PR.